app/main.py

```diff
@@ -1 +1,19 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> str:
+    cmd, current_file_name, new_path = command.split()
+    new_dirs, new_file_name = os.path.split(new_path)
+    if cmd != "mv":
+        return "Input correct command"
+    if len(new_dirs) > 1:
+        try:
+            os.makedirs(os.path.join(new_dirs))
+        except FileExistsError:
```

Comment:
Use exist_ok=True for makedirs. Because for example if I have already created directory but I need move file in this directory